commands/utils/utils.go

```diff
@@ -37,6 +38,14 @@ func (e *ErrMissingConfig) Error() string {
 	return fmt.Sprintf("config file is missing: %s", e.missingReason)
 }
 
+type OutputWriter interface {
```

Comment:
Let's add a description for each of the functions in this interface.

Reply:
done